src/graphql/fragments/loanCardFields.graphql

```diff
@@ -56,6 +56,12 @@ fragment loanCardFields on LoanBasic {
 	lenders(limit: 0) {
 		totalCount
 	}
+	
+	# for loan-progress component and lend cta component
+	unreservedAmount @client
```

Comment:
We don't want to add these fields are ALL loan cards, just the recommended. This is a shared fragment that applies in many app components.